src/gui/static/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,89 @@
+import { NgModule } from '@angular/core';
+import { WalletsComponent } from './components/pages/wallets/wallets.component';
+import { SendSkycoinComponent } from './components/pages/send-skycoin/send-skycoin.component';
+import { RouterModule } from '@angular/router';
+import { PendingTransactionsComponent } from './components/pages/settings/pending-transactions/pending-transactions.component';
+import { OutputsComponent } from './components/pages/settings/outputs/outputs.component';
+import { BlockchainComponent } from './components/pages/settings/blockchain/blockchain.component';
+import { BackupComponent } from './components/pages/settings/backup/backup.component';
+import { NetworkComponent } from './components/pages/settings/network/network.component';
+import { BuyComponent } from './components/pages/buy/buy.component';
+import { TransactionListComponent } from './components/pages/transaction-list/transaction-list.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin/pullRequest?prid=7163600)
```suggestion
import { TransactionListComponent } from "./components/pages/transaction-list/transaction-list.component";
```